server/lib/notifications/agents/gotify.ts

```diff
@@ -42,7 +47,8 @@ class GotifyAgent
     payload: NotificationPayload
   ): GotifyPayload {
     const { applicationUrl, applicationTitle } = getSettings().main;
-    let priority = 0;
+    const settings = this.getSettings();
+    const priority = settings.options.priority;
```

Comment:
A default value for priority is missing, for people that configured gotify notification before this was added.
```suggestion
    const priority = settings.options.priority ?? 1;
```